packages/gnuboy/makefile.patch

```diff
@@ -0,0 +1,5 @@
+--- makefile    2021-11-11 23:56:47.461415526 -0600
++++ makefile.mod      2021-11-11 23:58:59.361415432 -0600
+@@ -1,25 +0,0 @@
+.PHONY: linux
+.PHONY: clean
```

Comment:
This doesn't seem to be a valid patch file. Strongly recommending to create it with `diff` utility, especially if you don't know format internals.